readme.md

```diff
@@ -4,56 +4,28 @@
 
 [![Build Status](https://travis-ci.org/canjs/can-ndjson-stream.png?branch=master)](https://travis-ci.org/canjs/can-ndjson-stream)
 
+[![Build Status](https://saucelabs.com/browser-matrix/can-ndjson-stream.svg)](https://saucelabs.com/beta/builds/270fe2884b8940f68684fae62c89c39e)
 
 
-## Usage
+Most web applications encounter problems of latency because they process data discretely instead of in streams. ndjsonstream() converts a ReadableStream of raw ndjson data into a ReadableStream of Javascript objects.
```

Comment:
`ndjsonstream()` might look nice in a code block and a link to ReadableStream would be good too.